### PR TITLE
Put docker manifests in artifact manager

### PR DESCRIPTION
**Pull request by mgrand**
manifests.rs now puts manifests in the artifact manager.

### CODE REVIEW

**Review by chb0github — APPROVED** [review]

File: pyrsia_node/src/node_manager/handlers.rs [hunk 1]
```diff
@@ -30,7 +30,7 @@ use std::str;
 const ART_MGR_DIR: &str = "pyrsia";
 
 lazy_static! {
-    static ref ART_MGR: ArtifactManager = {
+    pub static ref ART_MGR: ArtifactManager = {
```
Comment (efrisch) [inline pyrsia_node/src/node_manager/handlers.rs:33]: This looks suspiciously like a Singleton. `ART_MGR_INSTANCE`?

**Review by efrisch — DISMISSED** [review]
Looks reasonable to me. 👍

File: pyrsia_node/src/node_manager/handlers.rs (hunk 1 above)
Comment (mgrand) [reply pyrsia_node/src/node_manager/handlers.rs:33]: This is a statically allocated instance of an ArtifactManager. Access to this is not managed by an object, so technically it is not a singleton. This made it through the review process  before. All I did was make this public.

**Review by efrisch — APPROVED** [review]
👍

**Review by fishseabowl — APPROVED** [review]
this seems reasonable to me.

**Pull request merged**